pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1048,12 +1048,12 @@ public PulsarClient getReplicationClient(String cluster) {
                     clientBuilder.serviceUrl(
                             isNotBlank(data.getBrokerServiceUrl()) ? data.getBrokerServiceUrl() : data.getServiceUrl());
                 }
-                if (data.getProxyProtocol() != null && StringUtils.isNotBlank(data.getProxyServiceUrl())) {
```

Comment:
I think we need this `data.getProxyProtocol() != null` check, no?

Reply:
StringUtils.isNotBlank contains this check, if `data.getProxyProtocol()` is null, then return false

Reply:
they are two different settings. One is `getProxyProtocol` and the other one is `getProxyServiceUrl`

Reply:
oh, sorry, I will fix it.